tests/unittests/Foundation/NSProgressTests.mm

```diff
@@ -353,11 +373,15 @@ - (void)addParentTo:(NSProgress*)child {
 
     // Set kind
     [progress setKind:NSProgressKindFile];
+	ASSERT_OBJCEQ(NSProgressKindFile,[progress kind]);
     [progress setUserInfoObject:NSProgressFileOperationKindDownloading forKey:NSProgressFileOperationKindKey];
     NSString* localizedDescriptionWithKind = [progress localizedDescription];
     NSString* localizedAdditionalDescriptionWithKind = [progress localizedAdditionalDescription];
     ASSERT_OBJCNE(@"", localizedDescriptionWithKind);
-    ASSERT_OBJCEQ(@"", localizedAdditionalDescriptionWithKind); // Yes, this matches the reference platform
+    #if TARGET_OS_WIN32
+        ASSERT_OBJCEQ(@"", localizedAdditionalDescriptionWithKind); // Yes, this matches the reference platform
+    #endif
```

Comment:
Why is "this matches the reference platform" now inside a block that doesn't build on the reference platform?